src/services/paymentService.js

```diff
@@ -79,8 +75,8 @@ class PaymentService {
       }
       return session.payment_status;
     } catch (error) {
-        /* istanbul ignore next */
-        res.status(500).json({ error });
+      /* istanbul ignore next */
+      res.status(500).json({ error });
```

Comment:
same here. error: error.message

Reply:
Also this code belongs to @teerenzo may be I will try to reach on him

Reply:
yaep